app/src/test/java/nl/tudelft/bbw/BlockChainAPITest.java

```diff
@@ -82,52 +82,27 @@ public final void addContactToChainTest() throws HashException, BlockAlreadyExis
      */
     @Test
     public final void revokeContactFromChainTest()
-            throws HashException, BlockAlreadyExistsException {
-        API.revokeContactFromChain(owner);
-        assertNotEquals(API.getBlocks(owner), list);
-    }
-
-    /**
-     * Test if adding to database yields a different chain
-     *
-     * @throws HashException               When creating a block results in an error
-     * @throws BlockAlreadyExistsException when adding a block results in an error
-     */
-    @Test
-    public final void addContactToDatabase() throws HashException, BlockAlreadyExistsException {
```

Comment:
I actually just copied your implementation of the tests of the previous API. But it is now changed with a better implementation.